### PR TITLE
fix(core): validate mandatory mfa

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Fix function `validateMandatoryBindMfa()`: the user only need any one of the factors, not all.

## Testing

Unit tests.

## Checklist

- [ ] `.changeset`
- [x] unit tests
- [ ] integration tests
- [ ] docs

### CODE REVIEW

**Review by xiaoyijun — APPROVED** [review]
LGTM

**Review by gao-sun — APPROVED** [review]
unblocking

File: packages/core/src/routes/interaction/verifications/mfa-verification.test.ts [hunk 1]
```diff
@@ -79,8 +79,11 @@ describe('validateMandatoryBindMfa', () => {
         validateMandatoryBindMfa(tenantContext, mfaRequiredCtx, interaction)
       ).rejects.toMatchError(
         new RequestError(
-          { code: 'user.missing_mfa', status: 422 },
-          { missingFactors: [MfaFactor.TOTP] }
+          {
+            code: 'user.missing_mfa',
+            status: 422,
+          },
+          { availableFactors: [MfaFactor.TOTP, MfaFactor.WebAuthn] }
```
Comment (simeng-li) [inline packages/core/src/routes/interaction/verifications/mfa-verification.test.ts:86]: define an SSOT for availableFactors?

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**